### DISCUSSION TITLE
Naomi/Naomi 2 service buttons

### DISCUSSION THREAD

**Discussion by antoniodesousa** [flyinghead/flycast#1002] [General]
I know that it's possible to access the service menu using the Flycast core in Retroarch, but I fail to see how to do the same thing in the Flycast standalone app. I don't see any option to enable the service buttons like in Retroarch or any possibility to map those buttons anywhere?

**Comment by flyinghead** [comment]
When mapping your controller make sure to select `Arcade Controls` in the upper right corner.
![image](https://user-images.githubusercontent.com/16261910/229301109-08801035-705b-4899-8f62-214ec1b1d4eb.png)
